src/main/java/org/lightcouch/View.java

```diff
@@ -438,14 +440,20 @@ public View key(Object... key) {
      *                 keys.
      */
     public View startKey(Object... startKey) {
-        this.startKey = getKeyAsJson(startKey);
-        uriBuilder.query("startkey", this.startKey);
+        this.startKey = getKeyAsJsonElement(startKey);
+        uriBuilder.query("startkey", this.startKey, true);
+        return this;
+    }
+
+    public View startKeyDocId(String startKeyDocId, boolean replace) {
```

Comment:
If we're hard-coding `replace=true` in the default method then adding this new method allows someone to specify `false` but is that necessary? Are there any circumstances where we would expect to have two of the same parameter for a view?